src/api/README.md

```
@@ -1,12 +1,18 @@
 # api
 
-서버와의 통신을 담당하는 함수 모음.
+서버(Cloudflare Pages Functions, `functions/api`)와의 통신을 담당하는 함수 모음.
 
 도메인별로 파일을 분리하며, 컴포넌트에서 fetch를 직접 호출하지 않는다.
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)